suite-native/formatters/src/components/FeeFormatter.tsx

```diff
@@ -18,7 +18,7 @@ export const FeeFormatter = ({ transaction }: FeeFormatterProps) => {
         () =>
             networkType === 'ethereum'
                 ? fromWei(transaction.ethereumSpecific?.gasPrice ?? '0', 'gwei')
-                : getFeeRate(transaction),
+                : transaction.feeRate || getFeeRate(transaction),
```

Comment:
I believe you can completely remove the legacy way using getFeeRate, check this comment https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/BasicTxDetails.tsx/#L276

Reply:
oh, nice. I will prepare fixup then. 👍